specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-01-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+          "readOnly": true
+        },
+        "status": {
+          "type": "string",
+          "description": "The current status of the asynchronous operation performed . For example, Running, Succeeded, Failed",
+          "readOnly": true
+        },
+        "properties": {
+          "title": "Properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagementGroupInfoProperties"
+        }
+      }
+    },
+    "HierarchySettings": {
+      "description": "Settings defined at the Management Group scope.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the settings object.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000/settings/default.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource.  For example, Microsoft.Management/managementGroups/settings.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the object. In this case, default.",
+          "readOnly": true
+        },
+        "properties": {
+          "title": "Properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/HierarchySettingsProperties"
+        }
+      }
+    },
+    "HierarchySettingsProperties": {
+      "description": "The generic properties of hierarchy settings.",
+      "type": "object",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The AAD Tenant ID associated with the hierarchy settings. For example, 00000000-0000-0000-0000-000000000000"
+        },
+        "requireAuthorizationForGroupCreation": {
+          "type": "boolean",
+          "description": "Indicates whether RBAC access is required upon group creation under the root Management Group. If set to true, user will require Microsoft.Management/managementGroups/write action on the root Management Group scope in order to create new Groups directly under the root. This will prevent new users from creating new Management Groups, unless they are given access."
+        },
+        "defaultManagementGroup": {
+          "type": "string",
+          "description": "Settings that sets the default Management Group under which new subscriptions get added in this tenant. For example, /providers/Microsoft.Management/managementGroups/defaultGroup"
+        }
+      }
+    },
+    "HierarchySettingsList": {
+      "description": "Lists all hierarchy settings.",
+      "properties": {
+        "value": {
+          "description": "The list of hierarchy settings.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HierarchySettingsInfo"
```

Comment:
>#/definitions/HierarchySettingsInfo [](http://example.com/codeflow?start=21&length=35)

This should be the same as returned by singleton GET, i.e. definitions/HierarchySettings.